src/query_parser/table_ref.py

```diff
@@ -0,0 +1,34 @@
+class TableInfo:
```

Comment:
Do you feel it's a good idea to move all the data classes to one place (**models** module)?